lib/openfoodfacts.dart

```diff
@@ -1022,4 +1022,46 @@ class OpenFoodAPIClient {
         '/' +
         filename;
   }
+
+  /// Unselect a product image.
+  ///
+  /// Typically, after that the openfoodfacts web page will _not_ show
+  /// the image as selected for this product x imagefield x language anymore.
+  /// Throws an exception if not successful.
+  /// Will work OK even when there was no previous selected product image.
+  static Future<void> unselectProductImage({
+    required final String barcode,
+    required final ImageField imageField,
+    required final OpenFoodFactsLanguage language,
+    final QueryType? queryType,
+  }) async {
+    final String id = '${imageField.value}_${language.code}';
+    final Uri uri = UriHelper.getUri(
+      path: 'cgi/product_image_unselect.pl',
+      queryType: queryType,
+      queryParameters: <String, String>{'code': barcode, 'id': id},
+    );
+
+    final Response response = await HttpHelper()
+        .doGetRequest(uri, userAgent: OpenFoodAPIConfiguration.userAgent);
```

Comment:
With a comma after the user agent it is probably a bit more readable.

Reply:
Good idea. I've just committed a refactoring.
It made me realize that it would probably be better to systematically include a `User user` parameter to methods where we impact the server database (not just read it).

Reply:
@M123-dev I have added a mandatory `User user` parameter in my recent `setProductImageAngle` and `setProductImageCrop` methods: does that qualify for a breaking change and the corresponding version upgrade? That would be a bit too generous.